app/app.js

```diff
@@ -71,13 +76,14 @@ app.use(assetPath, cacheHeaders);
 app.use(`${assetPath}vendor/v1`, express.static(path.join(__dirname, '..',
   'vendor', 'govuk_template_mustache_inheritance', 'assets')));
 
+app.use(`${assetPath}images`, express.static(path.join(__dirname, 'assets', 'images')));
+
 app.use(assetPath, express.static(path.join(__dirname, '..', 'dist', 'public')));
 
 app.use(helmet.noCache());
 
 app.use(`${basePath}/`, indexController);
 app.use(`${basePath}/`, cookieController);
```

Comment:
Do we still need the cookie controller?

Reply:
yeah we need a cookie page on all the apps, govuk requirement